### PR TITLE
Guild heavy barrel, jumpsuits and round 2 of plasma weapons balance

### PR DESCRIPTION
**Pull request by Dissentbay**
## About The Pull Request
<details>
<summary>

- solves a guild craft that was annyoing me, complete sidegrade
- guild jumpsuits are in, thanks to Ezo
- after some "heated" debate, I took a look at church's plasma competition, made them still the top of the game on plasma rifles, and made the "martyr belt" strategy less enticing
</summary>
<hr>

- excubitor is more inline with the antebellum, since it only works on church cells
- dominion and purger got more damage, and a bit more shots, and a small change in material cost
- cassad got put inline with the nerfs
- all plasma pistols got their power cost upped, also to be more inline with the nerfs
- psy weapons also got the nerf hammer, all to be more inline
- martyr got more inline with their class, the holdout pistols, more shots, way less damage

	
<hr>
</details>

## Changelog
:cl:
add: Guild emergency jumpsuits to loadout (by @Ezoken)
tweak: Heavy barrel recipes does not need a forged barrel, but all the steps to make a forged barrel were added in it's place (zero sum)
balance: nearly all plasma weapons had their stats changed(charge cost, damage and penetration), waiting on community feedback for further changes (hydrogen was untouched due to complexity, once the others get better balanced, they will too)

/:cl:

### CODE REVIEW

**Comment by Dissentbay** [comment]
I will leave for round 3 a revision on the excubitor X antebellum (in case it's needed), and make the blunderbuss viable

File: code/modules/projectiles/guns/energy/plasma/cassad.dm [hunk 1]
```diff
@@ -35,7 +35,7 @@
 	fire_sound = 'sound/weapons/energy/pulse.ogg'
 	projectile_type = /obj/item/projectile/beam/pulse
 	sel_mode = 1
-	charge_cost = 20 //40 shots per high medium-sized cell
+	charge_cost = 90 //40 shots per high medium-sized cell
```
Comment (Trilbyspaceclone) [inline code/modules/projectiles/guns/energy/plasma/cassad.dm:38]: please update the commit

Comment (Dissentbay) [reply code/modules/projectiles/guns/energy/plasma/cassad.dm:38]: will do

**Comment by Dissentbay** [comment]
on round 3 church armor also got added

**Pull request merged**